snippets/card-product.liquid

```diff
@@ -330,7 +330,10 @@
                   >
                     {% render 'icon-close' %}
                   </button>
-                  <div id="QuickAddInfo-{{ card_product.id }}" class="quick-add-modal__content-info"></div>
+                  <div
+                    id="QuickAddInfo-{{ card_product.id }}"
+                    class="color-{{ section.settings.color_scheme }} quick-add-modal__content-info"
```

Comment:
I don't think it needs to be added here, does it ? This is for the `standard` quick add which doesn't have any color scheme issues at the moment. When I check on the storefront, I see two color schemes class applied: 
![](https://screenshot.click/22-43-9b1pu-w7udg.png)